inc/classes/class-rocket-database-optimization.php

```diff
@@ -98,24 +110,63 @@ public function database_optimization_scheduled() {
 		}
 	}
 
+	/**
+	 * Database Optimization cron callback
+	 *
+	 * @since 3.0.4
+	 * @author Remy Perona
+	 */
+	public function cron_optimize() {
+		$items = [];
+
+		foreach ( array_keys( $this->options ) as $key ) {
```

Comment:
Just for fun, I think this will do the same: `$items = array_filter( array_keys( $this->options ), 'get_rocket_option' );` :)